### PR TITLE
Om 100 - bug fixes in master-cluster-view, alerts-view and unique-data-view

### PR DESCRIPTION
**Pull request by mphanias**
unique-data view - fixed bugs 1) unique-data-bytes are not displayed when geo-location is added and 2) query to honour selected namespaces
and 3) added a historical time series panel
alerts view - fixed display colour and show alerts specific to selection criteria
multi-cluster-view: fixed Grafana version to 9.x, XDR and Latencies point to respective dashboards (instead of cluster-view dashboard). added enhancement to filter using alert-severity

### CODE REVIEW

**Comment by mphanias** [comment]
action file python-lint.yml: moved python library to check-jsonschema as (existing jsonschema is having issues while installing and may get deprecate in future)

**Review by hev — APPROVED** [review]
Tested the dashboards. Links look good. Didn't notice much changed in AlertsView and unique data but worked fine.

File: config/grafana/provisioning/datasources/all.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ datasources:
     # orgId: 1
     url: http://alertmanager:9093
     editable: true
-    isDefault: false
```
Comment (hev) [inline config/grafana/provisioning/datasources/all.yaml:18]: This is setting alert manager as default instead of Aerospike Prometheus

**Review by hev — CHANGES_REQUESTED** [review]
See detail about default data source.

**Review by sunilvirus — APPROVED** [review]
Offline review comments addressed

File: config/grafana/provisioning/datasources/all.yaml (hunk 1 above)
Comment (sunilvirus) [reply config/grafana/provisioning/datasources/all.yaml]: Addressed.

**Pull request merged**